patterns/pricing.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Title: Pricing
+ * Slug: twentytwentyfour/pricing
+ * Categories: call-to-action
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|40","left":"var:preset|spacing|40","right":"var:preset|spacing|40"}}},"backgroundColor":"base-2","layout":{"type":"constrained"}} -->
```

Comment:
The group should have a border radius of 16 px:
 https://www.figma.com/file/AlYr03vh4dVimwYwQkTdf6/Twenty-Twenty-Four?node-id=492%3A4276&mode=dev

Reply:
Thanks for catching that. I'll add it.